wp-content/themes/core/components/section_nav/css/section-nav.pcss

```diff
@@ -0,0 +1,218 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Section Nav
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-section-nav {
+	border: 1px solid var(--color-border);
+
+	@media (--viewport-small) {
+		overflow: hidden;
+		border: none;
+	}
+
+	/* CASE: the more menu is in use. */
+	&.c-section-nav--more-initialized {
```

Comment:
💡 
I've see this pattern al throughout our code. Any reason we don't include this in the naming like:

`&.c-section-nav--more-case-initialized`

I've see other patterns like this that say things like `js` in the name so you know this modifier is managed by something else. Seeing that `case` would be a flag that this is changed by some logic somewhere based on a case.

Reply:
The [BEM syntax itself](http://getbem.com/introduction/) (the double-dash) is intended to inform you that this is a conditional case or a "modifier". We just try to add CSS comments whenever we use them to to clarify in human-readable format that the conditional modifier is.

Reply:
Totally makes sense. I'm just thinking in cases of "What is managing this class". Maybe it's not important, but in cases where we're adding/removing classes with PHP/JS it certainly can be tricky to reverse engineer things.